### PR TITLE
3.1 blablabla

### PR DESCRIPTION
**Pull request by bcisneros**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,8 @@ https://spa.myservername.com/sample-test-case-template-with-test-case-examples#D
 
 ## Unidad 3: Revisión de estándares ISO/IEC/IEEE
 ### 3.1 Serie 9000 (ISO 9000, ISO 9001)
+[Ver presentacion](3-1-iso-9000.txt)
+
```
Comment (CarlosMoriel) [inline README.md:161]: No debe quedar vacia esta linea

File: README.md [hunk 2]
```diff
@@ -157,6 +157,8 @@ https://spa.myservername.com/sample-test-case-template-with-test-case-examples#D
 
 ## Unidad 3: Revisión de estándares ISO/IEC/IEEE
 ### 3.1 Serie 9000 (ISO 9000, ISO 9001)
+[Ver presentacion](3-1-iso-9000.txt)
```
Comment (CarlosMoriel) [inline README.md:160]: Test

**Review by CarlosMoriel — CHANGES_REQUESTED** [review]

File: 3-1-iso-9000.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Here is the content of my presentation about ISO-9000
```
Comment (CarlosMoriel) [inline 3-1-iso-9000.txt:1]:
```suggestion
Here's  the content of my presentation about ISO-9000
```

**Review by CarlosMoriel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bcisneros) [reply README.md]: Done

**Review by CarlosMoriel — APPROVED** [review]

**Comment by bcisneros** [comment]
Closing this PR due is just an example

**Pull request closed without merging**